pages/docker_compose.md

```diff
@@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows, your Docker daemon has only l
 
 </div>
 
+
+If you encounter such an error `npm ERR! Error: EACCES: permission denied` when installing JHipster UML (or any unbundled package), your container may not have sudo (for instance, sudo isn't bundled with Ubuntu Xenial).
+A solution would be to do:
```

Comment:
I usually point people to https://docs.npmjs.com/getting-started/fixing-npm-permissions - isn't this better to go to the official NPM documentation?